sdk/timeseriesinsights/azure-mgmt-timeseriesinsights/README.md

```diff
@@ -15,12 +15,6 @@ library.
 For a more complete set of Azure libraries, see the
 [azure](https://pypi.python.org/pypi/azure) bundle package.
```

Comment:
same as above - you might want to use replace all :)

Reply:
i think all the packages have this reference - should i update in all 73 packages?

Reply:
yes